### PR TITLE
chore(tooling): update labeler for tooling related PRs

### LINKED ISSUE #4342: Update labeler for tooling apps
Update labeler to add out labels based on our paths for:

- ts sdk
- explorer
- wallet
- dashboard
- ui kit
- apps backend

https://github.com/iotaledger/iota/blob/develop/.github/labeler.yml

### PR DESCRIPTION
**Pull request by evavirseda**
# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/4342

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -4,3 +4,18 @@ documentation:
...
+wallet:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet/**/*
+wallet-dashboard:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet-dashboard/**/*
+ts-sdk:
+  - changed-files:
+      - any-glob-to-any-file: sdk/typescript/**/*
+ui-kit:
```
Comment (thibault-martinez) [inline .github/labeler.yml]:
```suggestion
apps-ui-kit:
```

**Review by thibault-martinez — APPROVED** [review]

**Review by msarcev — APPROVED** [review]

File: .github/labeler.yml [hunk 2]
```diff
@@ -4,3 +4,18 @@ documentation:
...
+wallet:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet/**/*
+wallet-dashboard:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet-dashboard/**/*
+ts-sdk:
+  - changed-files:
+      - any-glob-to-any-file: sdk/typescript/**/*
+apps-ui-kit:
```
Comment (msarcev) [inline .github/labeler.yml:16]: Should we go `ui-kit` here for consistency ? :)

Comment (begonaalvarezd) [reply .github/labeler.yml:16]: `apps-ui-kit` is fine imo

File: .github/labeler.yml [hunk 3]
```diff
@@ -4,3 +4,18 @@ documentation:
...
       - any-glob-to-any-file: apps/explorer/**/*
+wallet:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet/**/*
+wallet-dashboard:
+  - changed-files:
+      - any-glob-to-any-file: apps/wallet-dashboard/**/*
+ts-sdk:
+  - changed-files:
+      - any-glob-to-any-file: sdk/typescript/**/*
```
Comment (begonaalvarezd) [inline .github/labeler.yml]: we should mark it as the TS SDK if it matches any of the packages inside
```suggestion
      - any-glob-to-any-file: sdk/**/*
```

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
we should also add the `tooling` label for any file & folder we own, the owned files and folders can be checked at the codeowners file

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**